neo4j-bolt/src/test/groovy/io/micronaut/neo4j/bolt/health/Neo4jHealthIndicatorSpec.groovy

```diff
@@ -29,25 +32,37 @@ import spock.lang.Specification
  * @since 1.0
  */
 class Neo4jHealthIndicatorSpec extends Specification {
+
     void "test neo4j health indicator"() {
         given:
-        ApplicationContext applicationContext = ApplicationContext.run(
-                'neo4j.uri':"bolt://localhost:${SocketUtils.findAvailableTcpPort()}",
-                'neo4j.embedded.ephemeral':true
-        )
+        Neo4jContainer neo4jContainer = new Neo4jContainer(DockerImageName.parse("neo4j:latest"))
```

Comment:
please use a tag which matches the driver version instead of `latest`

Reply:
There isn't a 5.11 docker image... I don't know that the docker image has the same versioning as the driver?

Reply:
@sdelamo there is no tag that matches driver version 5.11.0. Latest image is 4.4.24.